apps/chat-e2e/src/testData/api/fileApiHelper.ts

```diff
@@ -38,12 +41,26 @@ export class FileApiHelper extends BaseApiHelper {
     return decodeURIComponent(body.url);
   }
 
-  public async deleteFile(path: string) {
+  public async deleteFromAllFiles(path: string) {
     const url = `/api/${path}`;
     const response = await this.request.delete(url);
     expect(response.status(), `File by path: ${path} was deleted`).toBe(200);
   }
 
+  public async deleteFromSharedWithMe(path: string) {
+    const encodedPath = path.split('/').map(encodeURIComponent).join('/');
+    const url = '/api/share/discard';
+    const requestData = {
+      resources: [{ url: encodedPath }],
+    };
+    const response = await this.request.post(url, {
+      data: requestData,
+    });
+    expect(
+      response.status(),
+      `File by path: ${path} was deleted from "Shared with me"`,
+    ).toBe(200);
+  }
```

Comment:
please, add a line break after the function

Reply:
done